test/main_test.go

```diff
@@ -187,3 +191,54 @@ func showScenarios(features []*gherkin.Feature) {
 	}
 	mainLogger.Info("------------------ END SHOW SCENARIOS ------------------")
 }
+
+// This does just create a namespace, install the operators via subscription if crd is missing
+// and then delete the namespace
+// Should be deleted once https://issues.redhat.com/browse/KOGITO-1376 is solved
+func importDependentOperatorCrds() {
+	mainLogger := framework.GetMainLogger()
+	mainLogger.Infof("Install dependent operator crds if needed")
+	namespace := framework.GenerateNamespaceName("crd-install")
+
+	if err := framework.CreateNamespace(namespace); err != nil {
+		panic(err)
+	}
+
+	installErr := installDependentOperatorsIfNeeded(namespace)
+	if installErr != nil {
+		mainLogger.Errorf("Error installing dependent operators: %v", installErr)
+	}
+
+	if err := framework.DeleteNamespace(namespace); err != nil {
+		panic(err)
+	}
+
+	if installErr != nil {
+		panic(installErr)
+	}
+}
+
+// Should be deleted once https://issues.redhat.com/browse/KOGITO-1376 is solved
+func installDependentOperatorsIfNeeded(namespace string) error {
+	var operatorsToInstall []string
+	for operatorName := range framework.KogitoOperatorCommunityDependencies {
+		if installed, err := framework.IsCommunityOperatorCrdAvailable(operatorName); err != nil {
+			return err
+		} else if !installed {
+			operatorsToInstall = append(operatorsToInstall, operatorName)
```

Comment:
I understand this is a temporary workaround, so take into account if you want, but I'd say that you could install and wait for the dependency in the same loop instead.

Reply:
AFAIK the intention is to speed up the test by allowing several dependencies to be installed in parallel

Reply:
exactly. the `waitFor` loop is blocking.
What I can do also to speed it up a bit is waiting for CRD to be installed, we do not care here that the operator pod is running. This should be quite quicker then.

Reply:
updated, should be a bit faster

Reply:
Inside this loop could have the second loop though.

Reply:
done :)